spark/src/main/java/org/apache/iceberg/spark/SparkFilters.java

```diff
@@ -63,8 +63,8 @@ private SparkFilters() {
 
   private static final ImmutableMap<Class<? extends Filter>, Operation> FILTERS = ImmutableMap
       .<Class<? extends Filter>, Operation>builder()
-      .put(AlwaysTrue$.class, Operation.TRUE)
-      .put(AlwaysFalse$.class, Operation.FALSE)
+      .put(AlwaysTrue.class, Operation.TRUE)
+      .put(AlwaysFalse.class, Operation.FALSE)
```

Comment:
`AlwaysTrue$` is the class of the Scala object `AlwaysTrue` and is a subclass of `AlwaysTrue`. Instead of removing it, let's add both classes.

Reply:
I see, let me add both and test again.